phylopandas/seqio/read.py

```diff
@@ -158,23 +158,23 @@ def read_blast_xml(filename, **kwargs):
         blast_record = NCBIXML.read(f)
 
     # Prepare DataFrame fields.
-    data = {'accession': [],
-            'hit_def': [],
-            'hit_id': [],
-            'title': [],
-            'length': [],
-            'e_value': [],
-            'sequence': []}
+    accession = []
+    hit_def = []
+    hit_id = []
+    title = []
+    length = []
+    e_value = []
+    sequence = []
 
     # Get alignments from blast result.
-    for i, s in enumerate(blast_record.alignments):
-        data['accession'] = s.accession
```

Comment:
Looks like I just forgot the append call. I think we could just call `.append` directly on these dict elements:

i.e. `data['accession'].append(s.accession)`

Otherwise their's no difference in our implementation except when the dictionary gets built.